bin/stashcp

```diff
@@ -172,20 +168,22 @@ def es_send(payload):
             url = "http://uct2-collectd.mwt2.org:9951"
             req = urllib2.Request(url, data=data, headers={'Content-Type': 'application/json'})
             f = urllib2.urlopen(req)
-            response = f.read()
+            f.read()
             f.close()
-        except:
-            logging.error("Error posting to ES")
+        except urllib2.URLError as e:
```

Comment:
Note that this format requires what, python 2.7?  That means this would eliminate using the system python on RHEL6.